guides/common/modules/proc_creating-a-webhook.adoc

```diff
@@ -0,0 +1,43 @@
+[id="creating-a-webhook_{context}"]
+= Creating a Webhook
+
+.Procedure
+
+. In the {Project} web UI, click *Administer* > *Webhooks*.
+. Click *Create Webhook*.
+. Click *Subscribe to field* to reveal a list of available events.
+. Select an event from the list.
+. Enter a name.
+. Enter a target URL. When using the shellhooks plugin, the URL should be in the form of `https://{smartproxy.example.com}:{smartproxy_port}/shellhooks/my_script`.
```

Comment:
```suggestion
. Enter a target URL.
When using the shellhooks plugin, the URL should be in the form of `https://{smartproxy-example-com}:{smartproxy_port}/shellhooks/my_script`.
```

one sentence per line and the attribute uses dashes, not dots. See [base attributes L95](https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L95)